json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -206,3 +206,24 @@ protected function result() {
 		);
 	}
 }
+
+class Jetpack_JSON_API_Sync_Now_Endpoint extends Jetpack_JSON_API_Sync_Endpoint {
+	protected function result() {
+		$args = $this->input();
+
+		if ( ! isset( $args['queue'] ) ) {
```

Comment:
Should we default it to one of the queues instead of returning an error?

If we do decide to return an error here, I think that we're also supposed to specify an http status code as the 3rd argument. Perhaps `400` for bad input?